src/tribler-core/tribler_core/components/tag/community/tag_community.py

```diff
@@ -0,0 +1,54 @@
+import json
+import random
+from binascii import unhexlify
+
+from pony.orm import TransactionIntegrityError, db_session
+
+from ipv8.lazy_community import lazy_wrapper
+from tribler_core.components.metadata_store.db.store import MetadataStore
+from tribler_core.components.tag.community.tag_payload import TagMessage, TagModel
+from tribler_core.modules.tribler_community import TriblerCommunity
+
+GOSSIP_RANDOM_TAGS_COUNT = 10
+GOSSIP_RANDOM_PEERS_COUNT = 10
+
+
+class TagCommunity(TriblerCommunity):
+    """
+    Community for disseminating tags across the network.
+    """
+
+    community_id = unhexlify('042020c5e5e2ee0727fe99d704b430698e308d98')
+
+    def __init__(self, *args, metadata_store: MetadataStore = None,
+                 gossip_interval_for_tags_in_sec=5, **kwargs):
```

Comment:
**Suggestion:** Move this 5 seconds into a global constant (like `GOSSIP_RANDOM_PEERS_COUNT`).

Reply:
What is the reason for moving it to a global constant? 

If it passes to the constructor, I can customize class behavior in a natural way.

Example:

https://github.com/drew2a/tribler/blob/28c8c8c8965c4147a14814224f3cb411df3b2a13/src/tribler-core/tribler_core/components/tag/community/tests/test_tag_community.py#L33-L34

Reply:
Yes, it's possible to customise this number, but extracting this number to a constant can improve readability and shows a developer that this is a default setting. It is also more in line with our current code practices, see for example [here](https://github.com/Tribler/py-ipv8/blob/master/ipv8/community.py#L32) and [here](https://github.com/Tribler/tribler/blob/main/src/tribler-core/tribler_core/modules/remote_query_community/tests/test_eva_protocol.py#L27). I dislike the declaration of constants without any context since we had too many of them in the past, which made it very hard to understand what's going on.

If we agree that extracting default constants to a global variable is unnecessary, we should at least try to be consistent 👍 .

Reply:
Fixed by extracting it to the const `DEFAULT_GOSSIP_INTERVAL_FOR_TAGS_IN_SEC`